src/config/config.ts

```diff
@@ -292,8 +292,6 @@ export class Configuration {
       }),
     },
     ethereum: {
-      ethScanApiUrl: process.env.ETH_SCAN_API_URL,
-      ethScanApiKey: process.env.ETH_SCAN_API_KEY,
```

Comment:
Remove in Bicep (let's do it together 😉)

Reply:
let's do it together is a great idea 😉